providers/common.go

```diff
@@ -6,21 +6,21 @@ import (
 )
 
 const (
-	isForkKey = "is_fork"
-	aliasesKey = "aliases"
+	IsForkKey  = "is_fork"
```

Comment:
do this implies more changes in borges and in other providers (the changes in borges that I made yesterday should be overriden).